app/src/main/java/com/hedvig/app/feature/insurance/ui/InsuranceAdapter.kt

```diff
@@ -219,24 +298,133 @@ class InsuranceAdapter(
 
     companion object {
         private const val TRANSITION_NAME = "contract_card"
+
+        private const val CROSS_SELL = 1
+        private const val SUBHEADING = 2
     }
 }
 
```

Comment:
Would it make sense to extract these composables to their own class/classes to increase visibility?

Reply:
Classes are less commonly used in Compose, but we can definitely move them to their own files 🙂

Reply:
Yeah if they are tightly coupled with this adapter then it makes sense to have them here. But I think we should try to make composables as reusable as possible from the start, so that you dont end up with many different variants.